src/probe.c

```diff
@@ -97,7 +97,9 @@ void probe_set_swclk_freq(uint freq_khz) {
 static inline void probe_assert_reset(bool state)
 {
     /* Change the direction to out to drive pin to 0 or to in to emulate open drain */
+#ifdef PROBE_PIN_RESET
```

Comment:
This seems to be an attempt to make the use of `PROBE_PIN_RESET` optional. Is it worth extending that plumbing all the way back into `CMakeLists.txt`, like you've done with `PICOPROBE_WS2812` ?

Reply:
That was because when I was starting off looking at this with reference to somebody else's branch, I selected the Arduino board profile as a starting point. However that defines its LED to be on pin 6 which Picoprobe uses as a reset signal... please feel free to remove that conditional but I do note that most descriptions of wiring a Picoprobe make so mention of the reset signal so my own opinion is that it's worth making it optional.

Reply:
Thoughts @liamfraser ?